Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -177,6 +177,9 @@ static void applyGeometryPropertiesToLayer(CALayer *layer, const RemoteLayerTree
         layer.contentsScale = properties.contentsScale;
         layer.rasterizationScale = properties.contentsScale;
     }
+
+    if (properties.changedProperties & LayerChange::MasksToBoundsChanged)
```

Comment:
Why did this move?

Reply:
This was always needed (interaction regions need to be clipped too).
But I think the method's name caused confusion :)

Reply:
Unfortunate that we still can't tell what they are common *between* by the name. But this is less wrong.